server/README.md

```diff
@@ -0,0 +1,16 @@
+#  
+First launch:
+    Make sure you have docker and postgres installed.
+    Open terminal:
+    Type:
+    docker run --name postgres-db -e POSTGRES_PASSWORD=password -p 25432:5432 -d postgres
+Second launch:
+    Open terminal:
+    Type:
+    docker start postgres-db
+
+To open database:
+    Open terminal:
+    Type:
+    psql -h 127.0.0.1 -p 25432 postgres postgres
```

Comment:
Czyli - to wrzuć sobie do notatek (serio, rób je gdzieś, rozwiązywanie tych samych problemów powinno zajmować Ci istotnie mniej czasu za drugim razem), a tutaj niepotrzebne.